vendor/composer/autoload_static.php

```diff
@@ -61,6 +61,10 @@ class ComposerStaticInit22ddb69348c7ed922c96325249cef3d0
         array (
             'Psr\\Log\\' => 8,
```

Comment:
Sorry for confusion; I meant this entire file.
There are 3846 deletions on in this file ( vendor/composer/autoload_static.php ).
I think you likely need to do the following command to fix this:
composer dump-autoload -o

This if from http://www.open-emr.org/wiki/index.php/Composer#Update_Autoloader